lib/text/ui_text_displayer.js

```diff
@@ -199,6 +199,19 @@ shaka.text.UITextDisplayer = class {
     this.isTextVisible_ = on;
   }
 
+  /**
+   * @private
+   */
+  isElementUnderTextContainer_(elemToCheck) {
+    if (elemToCheck == null) {
+      return false;
+    }
+    if (elemToCheck == this.textContainer_) {
+      return true;
+    }
+    return this.isElementUnderTextContainer_(elemToCheck.parentElement);
```

Comment:
With only one parent element, I think this could be rewritten as a while loop and be slightly more efficient that way.

https://jsbench.me/ suggests it would be 10% faster, though admittedly I did put the target element 100 levels deep in my test case.

Setup:
```js
  const myContainer = document.createElement('div');
  let innerHTML = '<span id="myTarget"></span>';
  for (i = 0; i < 100; i++) {
    innerHTML = `<span>${innerHTML}</span>`;
  }
  myContainer.innerHTML = innerHTML;
  document.body.appendChild(myContainer);
```

Recursive:
```js
function v1(thing, ancestor) {
  if (thing == null) {
    return false;
  }
  if (thing == ancestor) {
    return true;
  }
  return v1(thing.parentElement, ancestor);
}

console.log(v1(myTarget, document.body));
```

Loop:
```js
function v2(thing, ancestor) {
  while (thing != null) {
    if (thing == ancestor) {
      return true;
    }
    thing = thing.parentElement;
  }
  return false;
}

console.log(v2(myTarget, document.body));
```